charts/app/templates/_helpers.tpl

```diff
@@ -0,0 +1,47 @@
+{{/*
```

Comment:
Om, I would love you to walk me through this file a bit, and how it relates to the other _helpers.tpl files. Does this need to be invoked in a special way? Cheers.

Reply:
these files are auto invoked by HELM during template formation before deployment, so for each component it creates reusable labels and name and et..